airbyte-webapp/src/components/connection/CatalogTree/next/StreamConnectionHeader.module.scss

```diff
@@ -5,16 +5,37 @@ $icon-size: 15px;
 .container {
   display: flex;
   flex-direction: row;
-  justify-content: space-between;
-  padding: variables.$spacing-lg calc(variables.$spacing-lg * 2);
+  padding: variables.$spacing-xl calc(#{variables.$spacing-lg} * 2) 0;
```

Comment:
Same suggestion as above. No need to use the calc since it will appear in css as `calc(10px * 2)` instead of `20px` when adding sass vars.

Reply:
Thanks!  For my own understanding.... the right time to use `calc` is when you're referring to something outside of the css vars as part of the expression (like `10%`)... but if you're just evaluating something with the variables and a static number, simply put the variable?